editor/src/main/com/mbrlabs/mundus/ui/modules/Outline.java

```diff
@@ -129,8 +130,10 @@ public void onSceneChanged(SceneChangedEvent sceneChangedEvent) {
 
     @Override
     public void onSceneGraphChanged(SceneGraphChangedEvent sceneGraphChangedEvent) {
-        Log.traceTag(TAG, "SceneGraph changed. Building scene graph.");
-        buildTree(sceneGraph);
+//        Log.traceTag(TAG, "SceneGraph changed. Building scene graph.");
```

Comment:
You're right, but that's a seperate issue.
It's even more confusing if the outline does not change at all, if a new game object + model component is added using the ModelPlacementTool.